mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -532,4 +532,38 @@ Feature: Requests tests
     And match response.requesterId == extUserId
     And match response.pickupServicePointId == servicePointId
     And match response.status == 'Open - Not yet filled'
-    And match response.patronComments == extPatronComments
\ No newline at end of file
+    And match response.patronComments == extPatronComments
+
+  Scenario: Move request to another item on the same instance
+    * def extItemId1 = call uuid1
+    * def extItemId2 = call uuid1
+    * def extItemBarcode1 = 'FAT-1041-IBC-1'
+    * def extItemBarcode2 = 'FAT-1041-IBC-2'
+    * def extUserId = call uuid1
+    * def extUserBarcode = 'FAT-1041-UBC'
+    * def extRequestId = call uuid1
+    * def extRequestType = 'Page'
+    * def extMoveRequestId = call uuid1
+
+    # post first and second items in the same instance
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId1), extItemBarcode: #(extItemBarcode1) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId2), extItemBarcode: #(extItemBarcode2) }
+    # post an user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId), extUserBarcode: #(extUserBarcode), extGroupId: #(fourthUserGroupId) }
+    # post a request for the first item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostRequest') { requestId: #(extRequestId), itemId: #(extItemId1), requesterId: #(extUserId), extRequestType: #(extRequestType), extInstanceId: #(instanceId), extHoldingsRecordId: #(holdingId) }
+
+    # post a move request and verify that request moved to second item
+    * def moveRequestEntityRequest = read('classpath:vega/mod-circulation/features/samples/request/move-request-entity-request.json')
+    * moveRequestEntityRequest.id = extMoveRequestId
+    * moveRequestEntityRequest.destinationItemId = extItemId2
+    * moveRequestEntityRequest.requestType = extRequestType
```

Comment:
What about send those parameters directly like we do above -> call read('classpath:vega/mod-circulation/features/util/initData.feature@PostRequest') { requestId: #(extRequestId), itemId: #(extItemId1)...
something like that

Reply:
I checked from UI that the move request body is different than request body, so that, I decided to use a different one

Reply:
I mean we would create ANOTHER method that just similar to "postRequest'